### PR TITLE
Decisions: add proposal context

### PR DESCRIPTION
**Pull request by rperez89**
Screen Shot 2022-03-28 at 8 33 17 PM
Screen Shot 2022-03-28 at 8 33 07 PM

This closes <https://github.com/1Hive/gardens/issues/541>

### CODE REVIEW

File: src/components/Garden/Preferences/EVMExecutor.js [hunk 1]
```diff
@@ -268,6 +276,13 @@ function EVMExecutor({ evmcrispr }) {
 
   return (
     <Box heading="App selector">
+      <Field label="Context">
```
Comment (0xGabi) [inline src/components/Garden/Preferences/EVMExecutor.js:279]: What do you think of including a hint with a message suggesting that could be a forum post link?

File: src/components/Garden/DecisionDetail/DecisionDetail.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import {
   BackButton,
```
Comment (0xGabi) [inline src/components/Garden/DecisionDetail/DecisionDetail.js:5]: What do you think of reusing the conviction proposals logic and including the forum comments related to the link if any?

**Review by 0xGabi — COMMENTED** [review]
Well done, I left a couple of suggestions.

File: src/components/Garden/Preferences/EVMExecutor.js (hunk 1 above)
Comment (rperez89) [reply src/components/Garden/Preferences/EVMExecutor.js:279]: sure

File: src/components/Garden/DecisionDetail/DecisionDetail.js (hunk 2 above)
Comment (rperez89) [reply src/components/Garden/DecisionDetail/DecisionDetail.js:5]: ah yeah good idea

**Review by tmoutinho — APPROVED** [review]
✌️

File: src/components/Garden/Preferences/EVMExecutor.js [hunk 3]
```diff
@@ -276,7 +277,14 @@ function EVMExecutor({ evmcrispr }) {
 
   return (
     <Box heading="App selector">
-      <Field label="Context">
+      <Field label={
+        <div css={`display: flex; align-items:center;`}>
+          <div>Context</div>
+          <div>
+            <Help hint="">The context could be for example the forum post discussion link</Help>
```
Comment (0xGabi) [inline src/components/Garden/Preferences/EVMExecutor.js:284]:
```suggestion
            <Help hint="">Information related to your vote. For example: brief description or link to forum discussion.</Help>
```

**Review by 0xGabi — APPROVED** [review]

File: src/components/Garden/Preferences/EVMExecutor.js (hunk 3 above)
Comment (rperez89) [reply src/components/Garden/Preferences/EVMExecutor.js]: nice one gabi

**Pull request merged**